src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -586,6 +586,10 @@
     "NAME": "Procrastination",
     "DESCRIPTION": "Unplayable. NL Starts each combat in your discard pile."
   },
+  "stsjorbsmod:ProdigalMemory": {
+    "NAME": "Prodigal Memory",
+    "DESCRIPTION": "Draw 3 cards and lose !M! max HP. NL stsjorbsmod:Exert if not played."
```

Comment:
Wording on the "exert if not played" should probably be a little closer to how Burn/Shame/Regret/etc are worded. They all use `At the end of your turn, <do a thing>.` to make it clear when exactly the effect happens, but they're also unplayable, so I do think it's good for us to include some sort of conditional wording too. Maybe something like `If this card is in your hand at the end of your turn, stsjorbsmod:Exert.`?